cli/internal/providers/datacatalog/state/customtype.go

```diff
@@ -116,7 +117,8 @@ func (args *CustomTypeArgs) FromCatalogCustomType(from *localcatalog.CustomType,
 	args.Name = from.Name
 	args.Description = from.Description
 	args.Type = from.Type
-	args.Config = from.Config
+	args.Config = make(map[string]any)
+	maps.Copy(args.Config, from.Config)
```

Comment:
When we applied the GetResourceGraph twice, due to the Config which was shared, it started erroring out and the reason was not copying the map